vocode/streaming/agent/base_agent.py

```diff
@@ -326,7 +326,7 @@ def call_function(self, function_call: FunctionCall, agent_input: AgentInput):
                 agent_input.conversation_id,
                 params,
             )
-        event = self.interruptible_event_factory.create(action_input)
+        event = self.interruptible_event_factory.create(action_input, is_interruptible=action.is_interruptible)
```

Comment:
EDITOR: Why wasn't this passed before? Was this a bug or was I misunderstanding something?